src/HybridSystems.jl

```diff
@@ -85,27 +85,34 @@ for f in [:state_property_type, :transition_property_type]
 end
 
 # System
-export statedim, stateset, inputdim, inputset, guard, assignment, target_mode, resetmap
+export statedim, stateset, inputdim, inputset, guard, assignment, mode, target_mode, resetmap
 """
     statedim(hs::HybridSystem, u::Int)
 
-Returns the dimension of the state space of the system at mode `u`.
+Returns the dimension of the state space of the system at discrete state `u`.
 """
 MathematicalSystems.statedim(hs::HybridSystem, u::Int) = statedim(hs.modes[u])
 
 """
     stateset(s::AbstractSystem, u::Int)
 
-Returns the set of allowed states of the system at mode `u`.
+Returns the set of allowed states of the system at discrete state `u`.
 """
 MathematicalSystems.stateset(hs::HybridSystem, u::Int) = stateset(hs.modes[u])
 
 """
-    target_mode(hs::HybridSystem, t)
+    mode(hs::HybridSystem, u::Int)
+
+Returns the mode dynamical system at discrete state `u`.
```

Comment:
```suggestion
Returns the mode of the dynamical system at discrete state `u`.
```

Reply:
Is there a difference between *mode* and *discrete state*?